server/game/gameSystems/ReadyResourcesSystem.ts

```diff
@@ -0,0 +1,41 @@
+import { AbilityContext } from '../core/ability/AbilityContext';
+import type { Card } from '../core/card/Card';
+import { CardType, Location, WildcardCardType } from '../core/Constants';
+import Player from '../core/Player.js';
+import * as EnumHelpers from '../core/utils/EnumHelpers';
+import { IPlayerTargetSystemProperties, PlayerTargetSystem } from '../core/gameSystem/PlayerTargetSystem.js';
+
+
+export interface IReadyResourcesSystemProperties extends IPlayerTargetSystemProperties {
+    amount?: number
```

Comment:
My thought would be to make the amount non-optional so that the card writer is required to provide a number. The reason just being that when I see:

`immediateEffect: AbilityHelper.immediateEffects.readyResources(),`

with no amount, it's not obvious to me that its default behavior is to ready 1 resource